### ISSUE TITLE
pdflatex error when checking package that uses 'Config/reticulate' on RStudio Server

### ISSUE THREAD

**Issue by fdetsch** [rstudio/rstudio#11933]
### System details

    RStudio Edition : Server
    RStudio Version : 2022.07.1 Build 554
    OS Version      : Ubuntu 22.04.1 LTS
    R Version       : 4.2.1

### Steps to reproduce the problem
I am using `Config/reticulate` in the `DESCRIPTION` file to manage Python dependencies in a non-public R package. The entry looks like this:

```r
Config/reticulate:
  list(
    packages = list(
      list(package = "numpy", pip = TRUE),
      list(package = "pandas", pip = TRUE)
    )
  )
```

R CMD check succeeds when working with the local RStudio Edition on an otherwise similarly structured Ubuntu OS. However, on the RStudio Server, checking the package fails here:

```sh
* checking PDF version of manual ...
 WARNING
LaTeX errors when creating PDF version.
This typically indicates Rd problems.
LaTeX errors found:
!pdfTeX error: pdflatex (file tcrm1000): Font tcrm1000 at 600 not found
 ==> Fatal error occurred, no output PDF file produced!
* checking PDF version of manual without index ... ERROR
* DONE
Status: 1 ERROR, 1 WARNING
```

The same happens when running `rcmdcheck::rcmdcheck()` via the RStudio Console on the server. By contrast, invoking R CMD check on the source package `.tar.gz` via the server terminal succeeds, as does `rcmdcheck::rcmdcheck()` when starting `R` directly in the terminal.

What fixes the problem on the RStudio Server is using single instead of double codes, i.e.

```r
Config/reticulate:
  list(
    packages = list(
      list(package = 'numpy', pip = TRUE),
      list(package = 'pandas', pip = TRUE)
    )
  )
```

, in which case the check succeeds and the above `pdflatex` error disappears. I would have expected this to work with double quotes as well, just like it does when running R in the local RStudio Edition &ndash; not least because this is the approach that's recommended in the **reticulate** docs on [automatic configuration of Python dependencies](https://rstudio.github.io/reticulate/articles/python_dependencies.html#automatic-configuration). 

Since this is bound to the RStudio Server, I am having a hard time providing a reproducible example. Let me know if you require a minimal test package that integrates R and Python code and raises the above error on the RStudio Server.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@fdetsch Thank you for raising the issue, and I appreciate your finding a workaround! A minimal project would be helpful. But first: @kevinushey do you think this might be an issue in `reticulate` or is it something specific to RStudio Server?

**Comment by kevinushey** [comment]
> !pdfTeX error: pdflatex (file tcrm1000): Font tcrm1000 at 600 not found

This seems like an issue with either fonts on the system or the LaTeX installation itself. See e.g.

https://tex.stackexchange.com/questions/267675/pdftex-error-pdflatex-file-ecbx0800-font-ecbx0800-at-600-not-found
https://github.com/rstudio/tinytex/issues/173

**Comment by fdetsch** [comment]
I created a little test package to play around with: https://github.com/fdetsch/testpypkg

Weird thing is, I checked the package on two other servers and no error was raised when using double quotes. So it seems like this must somehow be related to how the first server is configured. 

I haven't been able to solve the issue so far, though. Reinstalling `texlive-scripts` didn't work, neither did `cm-super`. The only thing that has changed lately on said server was an Ubuntu dist-upgrade from 20.04 to 22.04. Maybe something got mixed up in the process. Anyway, I'll stick with single quotes for the time being (although I wouldn't have expected this to make such a difference). Thanks for your input!
